app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,6 @@
 class PostsController < ApplicationController
+  load_and_authorize_resource
+
   def index
     @user = User.find(params[:user_id])
     @posts = @user.posts
```

Comment:
- [ ] There is something I recognized when I am creating a post with a different user all posts are going to the
the first user and this has to be addressed because now a user is authenticated after logging in